src/script/interpreter.h

```diff
@@ -71,6 +71,46 @@ enum
     // verify dummy stack item consumed by CHECKMULTISIG is of zero-length (softfork safe, BIP62 rule 7).
     SCRIPT_VERIFY_NULLDUMMY = (1U << 4),
 
+    // Require all signature checks to pass (softfork safe, replaces BIP62
+    // rule 7, and is not compatible with SCRIPT_VERIFY_NULLDUMMY). It is
+    // still possible for the non-VERIFY forms of CHECKSIG or
+    // CHECKMULTISIG to be executed and return false, but you MUST do so
+    // by passing in an empty (OP_0) signature. The way this is
+    // implemented is different for the two opcodes:
+    //
+    // CHECKSIG and CHECKSIGVERIFY will abort script validation with error
+    // code SCRIPT_ERR_FAILED_SIGNATURE_CHECK if the signature validation
+    // code returns false for any reason other than the passed-in
+    // signature being the empty.
```

Comment:
Typo? Probably fine, but I would expect `signature being empty` or `signature being the empty OP_0 value` or something.

Reply:
Typo. It was "the empty string", but I like just "empty" better anyway. Fixed.